mappings/net/minecraft/client/input/MouseHelper.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_4095 net/minecraft/client/input/MouseHelper
```

Comment:
`net/minecraft/client/Mouse` ?

Reply:
This has a lot less functionality than the `Mouse` class in newer versions since most of it is handled by `org/lwjgl/input/Mouse` and this basically just adds 3 helper methods and a little state. In LWJGL 3 all of the functionality needs to be handled by Minecraft itself. I'm also a little confused why upstream doesn't have `Mouse` in the `input` package.

Reply:
ok fair, ig leave it as it is